bodhi/templates/update.html

```diff
@@ -535,6 +535,8 @@
             % endif
           % elif update.meets_testing_requirements and getattr(update.request, 'description', None) != 'stable':
             <a id='stable' class="btn btn-sm btn-success"><span class="glyphicon glyphicon-circle-arrow-right"></span> Push to Stable</a>
+          % elif update.stable_karma not in (0, None) and update.karma >= update.stable_karma and not update.autokarma and getattr(update.request, 'description', None) != 'stable':
```

Comment:
@kkofler Because this is for `manually push to stable is based of stable karma threshold` here. We don't want `autokarma` to be enabled here, if that happens I am not sure whether there would be any difference between `Autopush` and this feature. 
There have been two ways before i.e, Manually push to stable based on time(14 days in testing) and Autopush based on stable karma threshold.